stable/ibm-microclimate/RELEASENOTES.md

```diff
@@ -123,6 +111,7 @@ For detailed installation instructions go to https://microclimate-dev2ops.github
 
 | Chart | Date | Kubernetes Version Required | Image(s) Supported | Breaking Changes | Details |
 | ----- | ---- | ------------ | ------------------ | ---------------- | ------- |
+| 1.7.0 | Octiber 12, 2018 | 1.11.0  | 1810 | None | Various fixes and improvements |
```

Comment:
please fix typo in October with next release & be descriptive in the details -- this is all user will have to see what function previous version provided.